ambry-mysql/src/main/resources/AmbryContainerStorageStats.ddl

```diff
@@ -64,3 +64,52 @@ CREATE TABLE IF NOT EXISTS AggregatedAccountReportsMonth
     month VARCHAR(25) NOT NULL
 )
 CHARACTER SET utf8 COLLATE utf8_bin;
+
+/**
+ * This table is created to keep a list of partition class names. In main cluster, we have max-replicas-all-datacenters.
+ * In video cluster, we have video-311. For these two classes, we would have rows below:
+ * -------------------------------------------------
+ * clusterName  |id  |name                         |
+ * -------------------------------------------------
+ * Ambry-Prod   |1   |max-replicas-all-datacenters |
+ * Ambry-Video  |1   |video-311                    |
+ * -------------------------------------------------
+ */
+CREATE TABLE IF NOT EXISTS PartitionClassNames
+(
+    clusterName VARCHAR(25) NOT NULL,
+    id SMALLINT NOT NULL,
+    name VARCHAR(45) NOT NULL,
```

Comment:
I would call this partitionClassName.
Also, does this support multiple partition classes per cluster (which we may have in the future)?

Reply:
I think `name` is fine. It's table `PartitionClassNames` so we already know it should be partition classes' names.